modules/virt-hot-plugging-a-virtual-disk.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+//
+// * virtual_machines/virtual_disks/virt-hot-plugging-virtual-disks.adoc
+
+[id="virt-hot-plugging-a-virtual-disk_{context}"]
+= Hot-plugging a virtual disk
+
+Hot-plug virtual disks that you want to attach to a virtual machine instance (VMI) while a virtual machine is running.
+
+.Prerequisites
+* You must have a running virtual machine to hot-plug a virtual disk.
+* You must have at least one data volume or persistent volume claims (PVC) available for hot-plugging.
+
+To hot-plug a virtual disk using the CLI, use the `addvolume` command with `virtctl`.
+
+.Procedure
+
+. Run the following command:
+
+----
+$ virtctl addvolume <virtual-machine|virtual-machine-instance> --volume-name=<datavolume|PVC> [--persist] [--serial=<label-name>]
```

Comment:
Consider breaking up this long command into two lines by using a backslash to show the line break.